doc/user_guide/statements/insert.md

```diff
@@ -0,0 +1,54 @@
+# INSERT 
+
+You can construct [`INSERT`](https://docs.exasol.com/sql/insert.htm) SQL statements using the `Insert` class.
+
+## Creating `INSERT` Statements
+
+Create an `INSERT` statement:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .values("value1","value2","value3");
+```
+
+Create an `INSERT` statement for specific fields:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .field("column1", "column2","column3")
+                .values("value1","value2","value3");
+```
+
+### Using Placeholders
+
+```java
+		insert.valuePlaceholder()
+```
```

Comment:
I think it would be great to give a complete example here. (With a prepared statement)

Reply:
Right, did just that. Thanks for the feedback.